--- a/node_modules/@parcel/packager-html/lib/HTMLPackager.js	2026-06-22 10:02:02.330652300 +0000
+++ b/node_modules/@parcel/packager-html/lib/HTMLPackager.js	2026-06-22 10:23:35.767937300 +0000
@@ -105,7 +105,7 @@
           dependency,
           fromBundle: bundle,
           toBundle: entryBundle,
-          relative: false
+          relative: true
         }).to : dependency.specifier,
         module: false
       };
